stdlib/Move.toml

```diff
@@ -0,0 +1,14 @@
+[package]
+authors = ["MoveFuns DAO contributors"]
+name = "movefuns"
```

Comment:
name = `movefuns-stdlib`?

Reply:
movefuns-stdlib or MovefunsStdlib ?

I see that the move language use MoveStdlib as the std package's name.

Reply:
MoveFunsStdlib follow MoveStdlib

Reply:
Done